### PR TITLE
[ADD] add antropismo filters in monitoring

### PR DESCRIPTION
**Pull request by GabrielMarianosf**
add antropismo filters in monitoring

### CODE REVIEW

**Comment by GabrielMarianosf** [comment]
[Antropismo Consolidado](https://trello.com/c/xGyKeyCM/6-antropismo-consolidado)

**Comment by GabrielMarianosf** [comment]
[Antropismo Consolidado](https://trello.com/c/xGyKeyCM/6-antropismo-consolidado)

File: pages/index/monitoring.vue [hunk 1]
```diff
@@ -13,6 +13,45 @@
 
         <ShowDialog />
 
+        <v-container class="pa-0">
+            <!-- <div class="tab-header justify-space-between">
```
Comment (dagnaldo) [inline pages/index/monitoring.vue:17]: please, remove unecessary comments.

File: pages/index/monitoring.vue [hunk 2]
```diff
@@ -13,6 +13,45 @@
...
+                        :group="group"
+                    />
+                </template>
+            </v-list>
+            <div v-if="$fetchState.pending">
+                <template v-for="i in 6">
+                    <v-skeleton-loader :key="i" type="text" class="mx-4 my-5" />
+                </template>
+            </div>
+            <!-- <v-container>
```
Comment (dagnaldo) [inline pages/index/monitoring.vue:41]: please, remove unecessary comments.

File: pages/index/monitoring.vue [hunk 3]
```diff
@@ -106,14 +155,29 @@ export default {
                 this.$store.commit('monitoring/setShowFeatures', value)
             },
         },
+        orderedSupportLayersGroups() {
+            return _.sortBy(this.supportCategoryGroupsAntropismo, 'order')
+        },
```
Comment (dagnaldo) [inline pages/index/monitoring.vue:160]: Missing blank line.

File: pages/index/monitoring.vue [hunk 4]
```diff
@@ -106,14 +155,29 @@ export default {
                 this.$store.commit('monitoring/setShowFeatures', value)
             },
         },
+        orderedSupportLayersGroups() {
```
Comment (dagnaldo) [inline pages/index/monitoring.vue:158]: Missing blank line.

File: pages/index/monitoring.vue [hunk 5]
```diff
@@ -106,14 +155,29 @@ export default {
...
+            },
+            set(value) {
+                this.$store.commit('supportLayers/setShowFeatures', value)
+            },
+        },
+
+        ...mapState('supportLayers', [
+            'supportCategoryGroupsAntropismo',
+            'loading',
+        ]),
```
Comment (dagnaldo) [inline pages/index/monitoring.vue:173]: Missing blank line.

File: store/supportLayers.js [hunk 6]
```diff
@@ -53,6 +53,17 @@ export const getters = {
         }
         return activeLayerIds
     },
+    activeLayerIds(state) {
```
Comment (dagnaldo) [inline store/supportLayers.js:56]: Missing blank line.

**Review by dagnaldo — CHANGES_REQUESTED** [review]

File: components/support/SupportLayerFiltersAntropismo.vue [hunk 7]
```diff
@@ -0,0 +1,221 @@
...
+                        !hasDoubleDate &&
+                        (layer_filter.filter_type === 'start_date' ||
+                            layer_filter.filter_type === 'end_date')
+                    "
+                >
+                    <v-col :key="layer_filter.filter_type">
+                        <BaseDateField
+                            v-model="filters[layer_filter.filter_type]"
+                            :label="$t('start-date-label')"
+                            :required="true"
```
[the remark this answers is not in the document]
Comment (GabrielMarianosf) [reply components/support/SupportLayerFiltersAntropismo.vue]: Yes, resolved.

File: components/support/SupportLayerFiltersAntropismo.vue [hunk 8]
```diff
@@ -0,0 +1,221 @@
...
+                    hasStartDate = true
+                } else if (layerFilter.filter_type === 'end_date') {
+                    hasEndDate = true
+                }
+            })
+
+            this.hasDoubleDate = hasStartDate && hasEndDate
+        }
+    },
+    mounted() {
```
[the remark this answers is not in the document]
Comment (GabrielMarianosf) [reply components/support/SupportLayerFiltersAntropismo.vue:175]: Resolved.

File: components/support/SupportLayerFiltersAntropismo.vue [hunk 9]
```diff
@@ -0,0 +1,221 @@
...
+        this.getFilterOptions()
+    },
+    watch: {
```
[the remark this answers is not in the document]
Comment (GabrielMarianosf) [reply components/support/SupportLayerFiltersAntropismo.vue:179]: Resolved.

File: components/support/SupportLayerFiltersAntropismo.vue [hunk 10]
```diff
@@ -0,0 +1,221 @@
...
+    computed: {
+        ...mapState('supportLayers', ['filterOptions']),
+    },
+
+    methods: {
+        populateTiOptions(cr) {
+            if (cr) this.$store.dispatch('supportLayers/getTiOptions', cr)
+            else this.filters.ti = null
+        },
+        filterLayer() {
```
[the remark this answers is not in the document]
Comment (GabrielMarianosf) [reply components/support/SupportLayerFiltersAntropismo.vue:195]: Resolved.

File: components/support/SupportLayerFiltersAntropismo.vue [hunk 11]
```diff
@@ -0,0 +1,221 @@
...
+            const filterInfo = {
+                id: this.layer.id,
+                filters: this.filters,
+            }
+            console.log(this.filters)
```
[the remark this answers is not in the document]
Comment (GabrielMarianosf) [reply components/support/SupportLayerFiltersAntropismo.vue]: Resolved.

File: components/support/SupportLayersGroupItemAntropismo.vue [hunk 12]
```diff
@@ -0,0 +1,175 @@
...
+            </v-list-item-avatar>
+            <v-list-item-content>
+                <v-list-item-title class="text-wrap">
+                    <span class="text-cursor">
+                        {{ layer.name }}
+                    </span>
+                </v-list-item-title>
+            </v-list-item-content>
+
+            <!-- <v-list-item-action v-if="disabledHeatmap">
```
[the remark this answers is not in the document]
Comment (GabrielMarianosf) [reply components/support/SupportLayersGroupItemAntropismo.vue]: Resolved.

File: pages/index/monitoring.vue (hunk 2 above)
Comment (GabrielMarianosf) [reply pages/index/monitoring.vue]: Resolved.

File: pages/index/monitoring.vue (hunk 4 above)
Comment (GabrielMarianosf) [reply pages/index/monitoring.vue:137]: Resolved.

File: pages/index/monitoring.vue (hunk 5 above)
Comment (GabrielMarianosf) [reply pages/index/monitoring.vue:153]: Resolved.

File: store/supportLayers.js (hunk 6 above)
Comment (GabrielMarianosf) [reply store/supportLayers.js:59]: Resolved.

**Review by dagnaldo — APPROVED** [review]

**Pull request merged**